src/typing_extensions.py

```diff
@@ -165,7 +165,7 @@
 
 
 class _Sentinel:
-    def __repr__(self):
+    def __repr__(self):  # pragma: no cover
```

Comment:
To not conflict with #657
```suggestion
    def __repr__(self):
```